optuna/cli.py

```diff
@@ -203,6 +208,12 @@ def get_parser(self, prog_name: str) -> ArgumentParser:
         )
         return parser
 
+    @deprecated(
```

Comment:
Though it's not a strong opinion, showing both the deprecated warning and [experimental warning](https://github.com/optuna/optuna/blob/master/optuna/dashboard.py#L249) might be noisy. How about deleting the experimental warning to clearly show the deprecation plan?

```
/usr/local/lib/python3.8/site-packages/cliff/command.py:187: FutureWarning: dashboard has been deprecated in v2.7.0. This feature will be removed in v4.0.0. See https://github.com/optuna/optuna/releases/tag/v2.7.0. Please use optuna-dashboard (https://github.com/optuna/optuna-dashboard) instead.
  return_code = self.take_action(parsed_args) or 0
/workspaces/optuna/optuna/dashboard.py:270: ExperimentalWarning: Optuna dashboard is experimental (supported from v0.1.0). The interface can change in the future.
  _show_experimental_warning()
```